heft-plugins/heft-lint-plugin/package.json

```diff
@@ -19,7 +19,8 @@
   },
   "dependencies": {
     "@rushstack/node-core-library": "workspace:*",
-    "semver": "~7.5.4"
+    "semver": "~7.5.4",
+    "typescript": "~5.8.2"
```

Comment:
Is this actually a runtime dependency now?

Reply:
Good call, updated in [1f31c7e](https://github.com/microsoft/rushstack/pull/5239/commits/1f31c7e70ef3aee918bb713a84d9fe0317f2a786)